apps/storefront/src/components/cart/cart-panels/CartInformationPanel.tsx

```diff
@@ -142,11 +210,63 @@ export const CartInformationPanel = ({
               })
             }
           >
-            <option value="MN">Minnesota</option>
+            <option value="Alabama">Alabama</option>
```

Comment:
suggestion: this could be cleaner to store options in a constants file and map through them here